src/ui/src/js/jstree.js

```diff
@@ -0,0 +1,69 @@
+/*
+ * jsTree
+ */
+
+import "jstree";
+
+function attach_jstree(selector) {
+  return $(selector)
+    .jstree({
+      sort: 1,
+      core: {
+        data: {
+          url: function (node) {
+            return node.id === "#"
+              ? `/projects/storage/`
+              : `/projects/storage/${btoa(node.id)}`;
+          },
+        },
+      },
+      plugins: ["checkbox", "sort", "types", "wholerow"],
+      sort: function (a, b) {
```

Comment:
What is it sorting, and aren't there any built in functions you could use instead?

Reply:
The old implementation came back unsorted.  If it’s required now by using the new endpoint I’ve yet to test.

This a jsTree plugin which takes a function to find out how to sort the file tree.

Reply:
I see. Does it sort between folders and files? If we still need it, it would be nice with a little comment saying what it does.

Reply:
Frankly, I need to take a deeper look.  Stack said that this solved the problem with the old implementation.